tpr/tpr.go

```diff
@@ -51,13 +49,14 @@ type Config struct {
 type TPR struct {
 	clientset kubernetes.Interface
 
-	name          string
-	group         string
-	version       string
-	description   string
-	qualifiedName string // name.group
+	name        string // see Config.Name
+	kind        string // see Config.Name
+	group       string // see Config.Name
+	version     string
+	apiVersion  string // apiVersion is group/version
+	description string
 
-	endpointList string
+	endpoint string
```

Comment:
Because this endpoint can be used to more operations than only listing resource, this will be changed in my next PR anyway.